Source/Diagnostics/Diagnostics.cpp

```diff
@@ -103,6 +103,57 @@ Diagnostics::BaseReadParameters ()
 
     bool species_specified = pp.queryarr("species", m_species_names);
 
+    // Prepare to dump rho per species
+    amrex::Vector<std::string> variables;
```

Comment:
Please add the following includes above:
```C++
#include <AMReX_Vector.H>

#include <string>
```

`variables` is a quite generic name, could we name this something like `species_variables`?
```suggestion
    amrex::Vector<std::string> species_variables;  // variable selection per species
```